packages/devtools_app/lib/src/screens/vm_developer/object_store_controller.dart

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
```

Comment:
can we create a subfolder in vm_developer for the object_inspector? there are starting to be quite a few files for this tool